README.md

```diff
@@ -207,4 +207,23 @@ see the [Scaling Lightning](https://github.com/scaling-lightning/scaling-lightni
 project.
 
 ## Docker
-If you want to run the cli in a containerized environment, see the docker set up docs [here](./docker/README.md)
\ No newline at end of file
+If you want to run the cli in a containerized environment, see the docker set up docs [here](./docker/README.md)
+
+## Contributing
+SimLN is open and welcomes contribution from the community. When creating a pull request, ensure that you adhere to the following:
+### Passing tests
```

Comment:
Can we squash this into a "PR Requirements" section?  With an emphasis on running these checks before you open a PR. 

Something like: 

Before you open a PR, please make sure the following pass: 
- {list of requirements}
 
These checks will also be enforced by the repository's CI, and only passing PRs will be reviewed.